src/System.Net.WebHeaderCollection/src/System/Net/HeaderInfo.cs

```diff
@@ -1,29 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Net
```

Comment:
The contract's public API surface area is a lot smaller than the public surface area of the full framework.